File: Easy/JavaScript/1748. Sum of Unique Elements.js
var sumOfUnique = function(nums) {
    let map = new Map();
    let count = 0;

    for(let i = 0; i < nums.length; i++){
        if(!map[nums[i]])
            map[nums[i]] = 1;
        else
            map[nums[i]]++;
    }

    for(let i = 0; i < nums.length; i++){
        if(map[nums[i]] == 1)
            count += nums[i];
    }
    
    return count;
};

File: Easy/Python/1748. Sum of Unique Elements.py
class Solution(object):
    def sumOfUnique(self, nums):
        used = {}
        count = 0

        for i in nums:
            used[i] = used.get(i, 0) + 1
        
        for key, value in used.items():
            if value == 1:
                count += key
        
        return count
